libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -507,4 +507,18 @@ class Strings {
 
         assertPrints(mixedColor, "brown&blue")
     }
+
+    @Sample
+    fun all() {
+        assertPrints("fatima".any(Char::isWhitespace), "false")
+        assertPrints("Ariya".all(Char::isLowerCase), "false")
```

Comment:
Let's have here two strings that have opposite properties. e.g:
```
assertPrints("London".any(Char::isWhitespace), "false")
assertPrints("Rio de Janeiro".any(Char::isWhitespace), "true")

assertPrints("London".all(Char::isLetter), "true")
assertPrints("Rio de Janeiro".all(Char::isLetter), "false")
```
In my opinion, it better shows how `all` and `any` differs. What do you think?